testHelper/bodyMatchers.go

```diff
@@ -5,41 +5,82 @@ package testHelper
 import (
 	"encoding/json"
 	"fmt"
+	"io"
 	"reflect"
 	"strings"
 	"testing"
 
 	"github.com/apimatic/go-core-runtime/https"
 )
 
-// RawBodyMatcher checks if the expectedBody is contained within the JSON response body.
+func setTestingError(test *testing.T, responseArg any, expectedArg any){
+	test.Errorf("got \n%v \nbut expected %v", responseArg, expectedArg)
+}
+// RawBodyMatcher compares the response body with the expected body via simple string checking. In case of Binary response, byte-by-byte comparison is performed. The expected body takes a URI path to a remote file to compare with the Binary response, which must be valid URI path.
```

Comment:
Why is the comment for `RawBodyMatcher` describing Binary type handling? I don't see any URI logic in the function. 

If you want to document how it will handle binary types and URIs, I recommended writing a test for it.

Reply:
this description is taken from online docs. Additionally for the URL content is removed as it is the responsibility of other function `isSameFile`